package.json

```diff
@@ -846,4 +846,4 @@
     "yargs": "^15.4.1",
     "zlib": "^1.0.5"
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Our editorconfig rules (https://github.com/elastic/kibana/blob/master/.editorconfig#L12-L13) actually call for the final newline to be removed, as this is how CLI tools and auto-formatting of package.json files normally works.